IDisposableAnnotations/MustDisposeAttribute.cs

```diff
@@ -0,0 +1,12 @@
+namespace IDisposableAnnotations
+{
+    using System;
+
+    /// <summary>
+    /// The return value should be disposed by caller.
```

Comment:
Probably better if we change `should` to `must` to be consistent with the attribute's name.